### PR TITLE
form step 3

### PR DESCRIPTION
**Pull request by MaiUbeid**
#81

### CODE REVIEW

File: src/components/Steps/PaymentStep/PaymentCard.jsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
+import React, { Component } from 'react';
```
Comment (alaa-yasin) [inline src/components/Steps/PaymentStep/PaymentCard.jsx:1]: I think there is no need for importing ``` {Component} ```

File: src/components/Steps/PaymentStep/PaymentCard.jsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+    return (
+      <div className="payment__step">
+        <h1 className="payment__title">Payment</h1>
+        <div className="payment__card">
+          <p className="payment-card__description">
+            We have secured your tickets. To receive your tickets please
+            complete your payment as soon as possible via one of our secure
+          </p>
+          <p className="payment-card__timer-description">
```
Comment (alaa-yasin) [inline src/components/Steps/PaymentStep/PaymentCard.jsx:34]: I guess this is related to the timer component

Comment (MaiUbeid) [reply src/components/Steps/PaymentStep/PaymentCard.jsx:34]: yup, but it just dummies data until now!

File: src/components/Steps/PaymentStep/PaymentCard.jsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  toggleDetails() {
+    const { showCard, showDetails } = this.state;
+    this.setState({ showDetails: !showDetails, showCard: !showCard });
+  }
+
+  render() {
+    const { showCard, showDetails } = this.state;
+
+    return (
+      <div className="payment__step">
```
Comment (alaa-yasin) [inline src/components/Steps/PaymentStep/PaymentCard.jsx:27]: I think it would be more readable if you created two components instead of one

**Pull request merged**